Makefile

```
@@ -88,6 +91,35 @@ docs: ## Legacy docs target alias
+		echo "pytest or testcontainers not installed; skipping integration-test."; \
+		echo "Install dependencies via: cd $(INTEGRATION_TESTS_DIR) && uv sync"; \
+		exit 0; \
+	fi; \
+	if ! "$(PYTHON)" -c 'import docker; docker.from_env().ping()' >/dev/null 2>&1; then \
+		echo "No Docker-compatible runtime reachable; skipping integration-test."; \
+		echo "Start Podman: podman system service --time=0 & export DOCKER_HOST=unix://\$$XDG_RUNTIME_DIR/podman/podman.sock"; \
+		exit 0; \
+	fi; \
+	cd $(INTEGRATION_TESTS_DIR) && "$(PYTHON)" -m pytest -m integration provisioning $(PYTEST_FLAGS)
+
+integration-command-test: ## Run cmd-mox-based command-contract tests
```

checkmake (0.3.2)

[warning] Target body for "integration-command-test" exceeds allowed length of 5 lines (10).

(maxbodylength)